assets/templates/authorize_sharing.html

```diff
@@ -2,7 +2,7 @@
 <html lang="{{.Locale}}">
   <head>
     <meta charset="utf-8">
-    <title>{{.Title}}</title>
+    <title>Cozy</title>
```

Comment:
Will it be ok with future customizations ?

Reply:
huh? That's odd. I don't remember changing that. I think this is a mistake. 
Good catch, eagle eye ! 👍